styles.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 800px) {
+    body {
+      background-color: lightpink;
+    }
+  }
+  
+  #main-doc{
+    position:reltive;
```

Comment:
Be careful of the typos in the style.css file. The **position property** applied here will not work because it is not correctly written. Kindly check the spelling for the code to work properly in **both** html and css files. You can add linters to your project so this problem is quickly solved. 
You can check and configure these linters provided by Microverse:
 https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md